scripts/tools-versions-darwin.txt

```diff
@@ -4,8 +4,8 @@ cmake=3.17.3
 ninja=1.9.0
 gperf=3.1;hash=258483ea2fd445d42d869d495504f960202fe0c3
 dtc=1.4.7;hash=0747cc9d6b50ac8a3b1b576e4509494f2258aef2
-gnuarmemb=9-2019-q4-major
-gnuarmemb-short=9-2019q4
+gnuarmemb=10-2020-q4-major
+gnuarmemb-short=10-2020q4
 ses=5.34a
 west=0.10.0
```

Comment:
we are using 0.10.1 now.  can you update in all files?

Reply:
should we add the zephyr-sdk to the specification? 
0.12.3

Reply:
> we are using 0.10.1 now. can you update in all files?

Sure.

> should we add the zephyr-sdk to the specification?
> 0.12.3

For which reason ?
The only officially supported toolchain for now is GNU Arm emb, not Zephyr SDK.
As well as Zephyr SDK is only for Linux.

If we should officially support Zephyr SDK, that would be a discussion to be taken outside of this PR, imho.

Reply:
> > we are using 0.10.1 now. can you update in all files?
> 
> Sure.

Done.
Minimal version is still `0.10.0` as there is no requirement for user to bump to `0.10.1`.
`0.10.1` contains an improvement that will allow easier transisition from master -> main upstream in Zephyr.

Reply:
yes, I agree